module/documents/activity/mixin.mjs

```diff
@@ -157,6 +157,7 @@ export default function ActivityMixin(Base) {
      *                                                 scaling is not allowed.
      * @property {object} spell
      * @property {number} spell.slot                   The spell slot to consume.
+     * @property {boolean} [triggerSubsequentActions]  If explicitly false, subsequent actions will not be triggered.
```

Comment:
Maybe just `subsequentActions`, then we can indicate the inferred default value without having to re-indent the other config options :)

```suggestion
     * @property {boolean} [subsequentActions=true]    Trigger subsequent actions defined by this activity.
```